sdks/python/apache_beam/io/bigquery.py

```diff
@@ -335,6 +336,24 @@ def __init__(self, table=None, dataset=None, project=None, query=None,
     self.validate = validate
     self.coder = coder or RowAsDictJsonCoder()
 
+  def display_data(self):
```

Comment:
This is a use case that I considered too. We could rely on the `mro()` method to obtain display data from super-classes - though we might not always want this (?). What do you think, @robertwb ?